docs/getting_started.md

```diff
@@ -118,10 +118,10 @@ kubectl describe pod busybox-curl-7bd5767999-spf7v
     Image: ghcr.io/open-feature/flagd:v0.4.4
     Port: 8014/TCP
     Host Port: 0/TCP
-    Args:
-      start
-      --uri
-      core.openfeature.dev/default/featureflagconfiguration-sample
```

Comment:
Please, keep it in the previous format. `kubectl describe` does not return yaml but a more human-readable output

Reply:
Good point, thank you :)